ember-browser-services/src/test-support/index.ts

```diff
@@ -21,7 +24,13 @@ type Fakes = {
 export function setupBrowserFakes(hooks: NestedHooks, options: Fakes): void {
   setupWindowMock(hooks);
 
-  hooks.beforeEach(function (this: TestContext) {
+  // Switched to 'any' from 'TestContext' due to awkward migration period from
+  // DT to built-in-types.
+  // I don't know if it's possible to support both fake test-helper types and real ones
+  // (simultaneously)
+  //
+  // Additionally, these types have no bearing on end-user behavior, so this is low risk.
```

Comment:
Thanks for the comment here - I agree! I'm personally a lot less strict with types when it comes to tests